sdcm/cluster.py

```diff
@@ -1783,7 +1783,24 @@ def scylla_setup(self, disks):
         """
         result = self.remoter.run('/sbin/ip -o link show |grep ether |awk -F": " \'{print $2}\'', verbose=True)
         devname = result.stdout.strip()
-        self.remoter.run('sudo /usr/lib/scylla/scylla_setup --nic {} --disks {}'.format(devname, ','.join(disks)))
+        if self.parent_cluster.params.get('workaround_kernel_bug_for_iotune'):
+            # related issue: https://github.com/scylladb/scylla/issues/5181
+            # a known kernel bug will cause scylla_io_setup fails in executing iotune.
+            self.remoter.run(
+                'sudo /usr/lib/scylla/scylla_setup --nic {} --disks {} --no-io-setup'.format(devname, ','.join(disks)))
+            script = dedent("""echo \"disks:
+  - mountpoint: /var/lib/scylla
+    read_iops: 540317
+    read_bandwidth: 1914761472
```

Comment:
where those numbers came from ?

Reply:
It came from a succeed gce instance (same instance type, same test).
The kernel bug doesn't occur all the time, so we can get some succeed gce instance.

I will update the commitlog to describe this.

Reply:
BTW, vladz agreed with this workaround (copy a good config from succeed instance)